instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/metrics/DurationHistogram.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.metrics;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.DoubleHistogram;
+import io.opentelemetry.context.Context;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.concurrent.ThreadSafe;
+
+/** A Histogram instrument that records durations */
+@ThreadSafe
+public class DurationHistogram {
+
+  private final DoubleHistogram delegate;
+
+  public DurationHistogram(DoubleHistogram delegate) {
```

Comment:
Could be package private
```suggestion
  DurationHistogram(DoubleHistogram delegate) {
```